src/darwin/Framework/CHIP/MTRCertificateInfo.mm

```diff
@@ -1,5 +1,5 @@
 /**
- *    Copyright (c) 2022 Project CHIP Authors
+ *    Copyright (c) 2023 Project CHIP Authors
```

Comment:
Should this be 2022-2023?

Reply:
Yikes, good catch!  I had looked through all the copyright diffs one more time, and _still_ missed one!